CMakeLists.txt

```diff
@@ -462,6 +462,19 @@ else()
 	RemoveFlag(CMAKE_C_FLAGS_MINSIZEREL CACHE "/DNDEBUG" "Default C flags for MinSizeRel")
 endif()
 
+if(ENABLE_MYSQL)
+	execute_process(COMMAND mysql_config --cflags OUTPUT_VARIABLE MYSQL_CFLAGS OUTPUT_STRIP_TRAILING_WHITESPACE)
+	string(REGEX REPLACE "-I" "" MYSQL_CFLAGS "${MYSQL_CFLAGS}")
+	string(REGEX REPLACE "-DNDEBUG" "" MYSQL_CFLAGS "${MYSQL_CFLAGS}")
+	execute_process(COMMAND mysql_config --libs OUTPUT_VARIABLE MYSQL_LIBS OUTPUT_STRIP_TRAILING_WHITESPACE)
+	add_subdirectory("src/modules/mariadbpp/")
+
+	# ignore this warning for mariadbpp submodule
+	if(CMAKE_CXX_COMPILER_ID MATCHES "Clang")
```

Comment:
Needs to be outside the `if(ENABLE_MYSQL)` conditional.

Should probably be `if("${CMAKE_CXX_COMPILER_ID}" MATCHES "Clang")`, but it's getting late so I'm not testing that tonight.

Reply:
Fixed the first part.

This is consistent with how clang is checked for elsewhere in the file though.